internal/monitors/collectd/postgresql/postgresql.go

```diff
@@ -55,12 +55,19 @@ const monitorType = "collectd/postgresql"
 //     - type: "gauge"
 //       valuesFrom:
 //       - "test"
+//       instancePrefix: "test"
 //  databases:
 //  - name: "test"
 //    username: "username2"
 //    password: "password2"
 //    queries:
 //    - "exampleQuery"
+//
+//
+//metricsToInclude:
```

Comment:
This needs some clarification as to why it is necessary.  I see that it has something to do with the `instancePrefix` but should this be a glob like `- gauge.test*` since _test_ is a prefix?

Reply:
Also this is a great example of why we need to get rid of the whitelist filtering and make it inherent to the monitor so that people wouldn't have to jump through these hoops to make a metric come through.

Reply:
@keitwb I've added a note about how the metric name is made. Could you take a look and let me know if the example is clear now?